test/browser/browser_asrouter_targeting.js

```diff
@@ -285,6 +285,18 @@ add_task(async function checkFrecentSites() {
   TopFrecentSitesCache.expire();
 });
 
+add_task(async function check_firefox_version() {
+  const message = {id: "foo", targeting: "firefoxVersion > 0"};
+  is(await ASRouterTargeting.findMatchingMessage({messages: [message]}), message,
+    "should select correct item when filtering by firefox version");
+});
+
+add_task(async function check_geoCountry() {
+  const message = {id: "foo", targeting: "geoCountry in ['US']"};
```

Comment:
I will do a try run before to confirm this expectation holds in our testing infrastructure but local builds seem to be `US`.

Reply:
Why not use `SpecialPowers.pushPrefEnv` to test this more robustly (probably with something other than `US`)